### ISSUE TITLE
Please package this as a Flatpak

### ISSUE THREAD

**Issue by fyrstormer** [Swordfish90/cool-retro-term#819]
I see this is available as an AppImage and as an Apple Disk Image. However, Flatpak is by far the most widely-supported format for sandboxed applications on Linux. Please package this as a Flatpak so it can be easily installed on distros that don't have AppImage supported out-of-the-box.

**Comment by luisbarrancos** [comment]
@fyrstormer an appimage file is an executable. If your distribution cannot run executables you have a bigger problem than not being able to run cool-retro-term.
Type ```file «your appimage app here».AppImage```, and you'll see something similar to ```ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.18, stripped```.
Chances are if you can't run them, they're not executable, so change their permissions, make them executable, e.g.: ```chmod 755 «your appimage app here».AppImage``` and off you go. If you need to run it from a desktop, create a ```*.desktop``` file in your MIME applications directory. There are plenty of resources showing you how to do that.

**Comment by fyrstormer** [comment]
Let me rephrase: It would be nice to be able to use Flatpak, and by extension any distro's app store which Flatpaks integrates with, to manage my installation of Cool Retro Term the same as if it were available in that distro's package repos. Rather than needing to set up anything manually.

**Comment by luisbarrancos** [comment]
@fyrstormer I understand the Flatpak experience would enhance the user experience, not only yours, in some distributions. Being the project open-source perhaps you could provide a Flatpak? The AppImage seems to be old and other users would greatly appreciate it as well.

**Comment by fyrstormer** [comment]
Unfortunately I don't know anything about packaging an application as a Flatpak. I'm a halfway decent Linux user, but I haven't had the free time to experiment with any developer tasks in Linux.
